apps/voting/app/src/vote-settings.js

```diff
@@ -1,8 +1,8 @@
 const voteSettings = [
   ['token', 'tokenAddress'],
   ['voteTime', 'voteTime', 'time'],
-  ['PCT_BASE', 'pctBase', 'number'],
-  ['supportRequiredPct', 'supportRequiredPct', 'number'],
+  ['PCT_BASE', 'pctBase', 'bignumber'],
+  ['supportRequiredPct', 'supportRequiredPct', 'bignumber'],
```

Comment:
We don't do any parsing of `'number'` declarations anymore, but I thought it'd be fitting to assign `'bignumber'` for numbers that are expected to be passed back as a string.

Reply:
I’m not sure it’s related, but I get some values like `pctBase` and `supportRequiredPct` as `Number` on a testing DAO (`p.aragonid.eth`), making BN.js throw an error when being instantiated.

We could make it work by converting the values:

```diff
diff --git a/apps/voting/app/src/App.js b/apps/voting/app/src/App.js
index bd2df99..0a56c58 100644
--- a/apps/voting/app/src/App.js
+++ b/apps/voting/app/src/App.js
@@ -264,8 +264,8 @@ export default observe(
         ...state,
 
         appStateReady,
-        pctBase: new BN(pctBase),
-        supportRequiredPct: new BN(supportRequiredPct),
+        pctBase: new BN(String(pctBase)),
+        supportRequiredPct: new BN(String(supportRequiredPct)),
         tokenDecimals: new BN(tokenDecimals),
 
         numData: {
@@ -284,11 +284,11 @@ export default observe(
                   data: {
                     ...data,
                     endDate: new Date(data.startDate + voteTime),
-                    minAcceptQuorum: new BN(data.minAcceptQuorum),
+                    minAcceptQuorum: new BN(String(data.minAcceptQuorum)),
                     nay: new BN(data.nay),
-                    totalVoters: new BN(data.totalVoters),
+                    totalVoters: new BN(String(data.totalVoters)),
                     yea: new BN(data.yea),
-                    supportRequiredPct: new BN(supportRequiredPct),
+                    supportRequiredPct: new BN(String(supportRequiredPct)),
                   },
                   numData: {
                     minAcceptQuorum:

```

But I don’t understand why? It comes as strings with other DAOs.

Reply:
@bpierre It's likely the cache, if you hadn't purged local storage before testing.

Reply:
It was the cache 😅 thanks!